react/modules/interpolateLink.ts

```diff
@@ -20,7 +22,9 @@ export default function interpolateLink(params: Params) {
       `{${namespace ? `${namespace}.${key}` : key}}`,
       'g'
     )
-    resolvedLink = resolvedLink.replace(regex, variables[key])
+
+    // we slugify the context value so we don't end up with invalid characters on the URL
+    resolvedLink = resolvedLink.replace(regex, searchSlugify(variables[key]))
```

Comment:
Should we do this to all values being passed? Can we just do this on brand?

Reply:
Was originally going to do that, but talked to @Klynger, and he said it would be a good idea to slugify all the parameters. We can do just with the brand for now, though

Reply:
Oh, we would need to do it for the specification properties as well

Reply:
I think it's safer